src/schedule-and-details/basic-section/BasicSection.scss

```diff
@@ -0,0 +1,36 @@
+.basic-section {
+  .basic-info-list {
+    display: flex;
+    justify-content: space-between;
+    margin: .75rem 0 1.75rem;
+    padding-inline-start: 0;
+    list-style-type: none;
+  }
+
+  .pgn__pageBanner-component {
+    background-color: $white;
+    border-radius: 6px;
+    box-shadow: 0 .0625rem .125rem rgba(0 0 0 / .15), 0 .0625rem .25rem rgba(0 0 0 / .15);
+  }
+
+  .pgn__pageBanner-content {
+    align-items: flex-start;
+    padding: 1.5rem;
```

Comment:
It might be a good idea to use a `$spacer` from Paragon for `1.5rem`. In this and other style files, you have created.

Spacer map
<img width="361" alt="image" src="https://github.com/raccoongang/frontend-app-course-authoring/assets/93188219/6922a8d5-8a4f-49ed-843b-b36d608148a5">

Usage `map-get($spacers, 3)`.

Reply:
I think that would be redundant, we've already used `rem`))